src/scripts/tray.js

```diff
@@ -24,6 +24,17 @@ const icons = {
 
 const _iconTray = path.join(__dirname, 'images', icons[process.platform].dir, icons[process.platform].icon || 'icon-tray.png');
 const _iconTrayAlert = path.join(__dirname, 'images', icons[process.platform].dir, icons[process.platform].iconAlert || 'icon-tray-alert.png');
+const _iconTrayTitleDot = path.join(__dirname, 'images', icons[process.platform].dir, icons[process.platform].iconTrayTitleDot || 'title/icon-tray-title-dot.png');
```

Comment:
Actually they are just examples. They do not look good. In linux they should be 44x38 px. Now they have 32x32